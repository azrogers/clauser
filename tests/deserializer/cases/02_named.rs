use clauser_macros::duplicate_keys;

#[duplicate_keys]
struct Test(Vec<String>);

fn main() {}
