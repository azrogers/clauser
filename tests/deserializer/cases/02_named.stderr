error: error at callsite #0 bytes(62..66): from_duplicate_key only supported on named fields
 --> tests/deserializer/cases/02_named.rs:3:1
  |
3 | #[duplicate_keys]
  | ^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `duplicate_keys` (in Nightly builds, run with -Z macro-backtrace for more info)
